1.8/administration/banning

```diff
@@ -18,6 +18,26 @@ To ban an IP address, just enter the IP address, or range of IP addresses using
 
 You can later delete any banned IP address by simply selecting the delete icon on the right hand side of the page for the corresponding IP address.
 
+## Banned Accounts
+
+The banning section is located under the Users & Groups tab and it allows you to ban registered users.
+
+#### Banning Accounts
+
+To ban registered user you need to fill following fields:
+
+[*] **Username** - The username of the account you wish to ban. Auto-complete is enabled on this field. (This field is necessary).
+[*] **Ban reason** - The reason to show the user explaining why they were banned.
+[*] **Ban Length** - The length of the ban (and when it will be lifted). (This field is necessary).
+
+#### Editing or Removing Ban for Account
+
+You can remove ban for some account by selecting **Lift** option under *Controls* section. Also, you can edit ban reason or expand ban by selecting **Edit** option under *Controls* section.
```

Comment:
This should be:
 You can remove a ban for a user by selecting the **Lift** option under the *Controls* section. Also, you can edit the ban reason or expand the ban by selecting the **Edit** option under the *Controls* section.